core/controllers/base_test.py

```diff
@@ -1839,6 +1839,12 @@ class MockHandlerWithDefaultGetSchema(base.BaseHandler):
                         'type': 'basestring'
                     },
                     'default_value': 'random_exp_id'
+                },
+                'apply_draft': {
+                    'schema': {
+                        'type': 'bool'
+                    },
+                    'default_value': 'false'
```

Comment:
This is not a valid default value for a boolean.

For something like this you could perhaps do "only set it to True if the apply_draft parameter is passed and it has a nonempty value". So the default is False, and "?apply_draft=" is also false, and the omission of the "apply_draft" key-value parameter is also False, but "?apply_draft={{any_nonempty_string}}" ends up being True.

Reply:
OK, yeah, fair enough. I read up on some of the ways people do it, and it is ambiguous in general. 

Let's go with the suggestion of "?apply_draft=false" being false and "?apply_draft=true" being true, where the 'true' and 'false' in the URL are case-insensitive. Anything else is false (this follows roughly the approach [Mailchimp](https://mailchimp.com/developer/release-notes/handling-boolean-query-parameters/) uses too). Sounds good?